test/remote/account_unlock_tests.js

```diff
@@ -186,6 +186,108 @@ TestServer.start(config)
   )
 
   test(
+    'unlock account with incorrect verify code',
+    function (t) {
+      var email = server.uniqueEmail()
+      var password = 'allyourbasearebelongtous'
+      var client
+      return Client.createAndVerify(config.publicUrl, email, password, server.mailbox)
+      .then(
+        function () {
+          return Client.login(config.publicUrl, email, password)
+        }
+      )
+      .then(
+        function (x) {
+          client = x
+        }
+      )
+      .then(
+        function () {
+          return client.lockAccount(email, password)
+        }
+      )
+      .then(
+        function () {
+          return client.resendAccountUnlockCode()
+        }
+      )
+      .then(
+        function () {
+          return server.mailbox.waitForCode(email)
+        }
+      )
+      .then(
+        function (code) {
+          return client.verifyAccountUnlockCode(client.uid, 'deadbeefbaadf00ddeadbeefbaadf00d')
+        }
+      )
+      .then(
+        function () {
+          t.fail('verifyAccountUnlockCode was expected to fail')
+        },
+        function (error) {
+          t.equal(error.code, 400)
+          t.equal(error.error, 'Bad Request')
+          t.equal(error.errno, 105)
+          t.equal(error.message, 'Invalid verification code')
+        }
+      )
+    }
+  )
+
+  test(
+    'unlock account with bad verify code',
+    function (t) {
+      var email = server.uniqueEmail()
+      var password = 'allyourbasearebelongtous'
+      var client
+      return Client.createAndVerify(config.publicUrl, email, password, server.mailbox)
+      .then(
+        function () {
+          return Client.login(config.publicUrl, email, password)
+        }
+      )
+      .then(
+        function (x) {
+          client = x
+        }
```

Comment:
same as https://github.com/mozilla/fxa-auth-server/pull/1002/files#r36325979 .
some tests have an extra then like here: https://github.com/mozilla/fxa-auth-server/pull/1002/files#diff-bec6d0b8b71c2e3fbd78548179467a41R108 some do not. should be consistent